stix_shifter_modules/security_advisor/stix_transmission/auth.py

```diff
@@ -25,4 +26,15 @@ def obtainAccessToken(self):
             return response.json()["access_token"]
 
         else:
-            raise Exception("Authorizaion Failed")
\ No newline at end of file
+            raise Exception("Authorizaion Failed")
+
+    def find_location(self, accountID, host):
+        if host == "":
+            adminApiUrl = os.getenv('ADMIN_API_URL', 'https://compliance.cloud.ibm.com/admin/v1')
+            headers = {"Authorization": "Bearer {0}".format(self.obtainAccessToken())}
+            resp = requests.get("{0}/accounts/{1}/settings".format(adminApiUrl, accountID), headers=headers)
+            current_location_id = resp.json()["location"]["id"]
```

Comment:
Is there any danger that either location or id won't be in the `resp` object? If so you'll want to handle this potential error.

Reply:
Done